### PR TITLE
Prevent the transistent db from blocking the upgrade process

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: docs/changelog.rst [hunk 1]
```diff
@@ -3,6 +3,7 @@ Changelog
 =========
 
 * :bug:`-` Fix empty asset location distribution for ETH2.
+* :bug:`-` The database with past profit and loss reports won't block the version upgrade process.
```
Comment (LefterisJP) [inline docs/changelog.rst:6]: mention that this is some edge cases

File: rotkehlchen/db/dbhandler.py [hunk 2]
```diff
@@ -310,7 +310,9 @@ def _run_actions_after_first_connection(self, password: str) -> None:
             if transient_version != ROTKEHLCHEN_TRANSIENT_DB_VERSION:
                 # "upgrade" transient DB
                 tables = list(cursor.execute('select name from sqlite_master where type is "table"'))  # noqa: E501
+                cursor.executescript('PRAGMA foreign_keys = OFF;')
```
Comment (LefterisJP) [inline rotkehlchen/db/dbhandler.py:313]: have you tested this?

Comment (yabirgb) [reply rotkehlchen/db/dbhandler.py:313]: Yes, with my database and finally got it working

**Review by LefterisJP — APPROVED** [review]
LGTM.

**Pull request merged**